src/main/scala/io/sdkman/changelogs/java/MandrelMigrations.scala

```diff
@@ -0,0 +1,36 @@
+package io.sdkman.changelogs.java
+
+import com.github.mongobee.changeset.{ChangeLog, ChangeSet}
+import com.mongodb.client.MongoDatabase
+import io.sdkman.changelogs.{
+  Mandrel,
+  Linux64,
+  MacOSX,
+  Version,
+  Windows,
+  removeVersion
+}
+
+@ChangeLog(order = "035")
+class MandrelVmMigrations {
+
+  @ChangeSet(
+    order = "001",
+    id = "001-add_mandrel_20_1_0_1",
+    author = "AlexKovynev"
+  )
+  def migrate001(implicit db: MongoDatabase) =
+    List(
+      Version(
+        candidate = "java",
+        version = "20.1.0.1-mandrel",
```

Comment:
Currently, the version just support 15 characters.

Reply:
I pushed it to 17 characters sometime ago for JDKs so this should be fine.

Reply:
Yep, the same length in SAP Jdk now. Stay the same